Lib/idlelib/pyshell.py

```diff
@@ -31,14 +30,13 @@
 import tokenize
 import warnings
 
-from idlelib import testing  # bool value
+import idlelib
```

Comment:
Move this also to __main__.  'idlelib' by itself is never used in main body of module.